classes/asserter/resolver.php

```diff
@@ -15,6 +15,7 @@ class resolver
 	protected $baseClass = '';
 	protected $namespaces = array();
 	private $analyzer;
+	private $resolved = array();
```

Comment:
Maybe a static attribute is better here, to shared the cache between all instances of the resolver (even if there is only one instance in atoum, in theory).